content/docs/nav.yml

```diff
@@ -1,89 +1,89 @@
-- title: Installation
+- title: Instalacija
   items:
     - id: getting-started
-      title: Getting Started
+      title: Prvi koraci
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Dodavanje React-a
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Nova React aplikacija
     - id: cdn-links
-      title: CDN Links
+      title: CDN linkovi
     - id: release-channels
       title: Release Channels
-- title: Main Concepts
+- title: Glavni koncepti
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Zdravo, svete
```

Comment:
I am not sure shall we translate this into Serb
@vaske what do you think ?

Reply:
I think it should remain as `Hello World`